### PR TITLE
Completed Accordion Slider

### PR DESCRIPTION
**Pull request by handasolo**
courtesy @vikramj74 and @h4xr.Known bug: Disaligns on first hover. Please, try to reproduce the bug and help in debugging, thank you.

### CODE REVIEW

**Comment by h4xr** [comment]
Unable to reproduce the misalignment bug on first hover.

**Comment by CuriousLearner** [comment]
Hi @Slayslot and @h4xr, Thanks for working on this section and making the PR.

I've reviewed your work and found the following things which should be rectified, before merging this PR.

1) The slider occupies more than 100% of the width with respect to the width occupied by other sections in the website. Please see the image below for reference:

slider

2) For size for md devices, the "HELP CHANGE THE WEB" section just breaks and text and image just overlaps. Please see the image below:

slider2

3) For md devices, the image for "THE WEB YOU WANT" section just vanishes. Please see the image below:

slider3

4) If someone tries to expand or contract the window, the content of the slider breaks, which is resolved only after first hovering, while this should not happen.

slider4

5) Also, I've noticed that more than half of each section is having hyper-link to respective sources and there are also links given separately on each card. Please follow either of the two, preferably, having a small link only. Also, please remove the default styling of underlining the text for the links.

6) As per a user, I've found that as soon as I hover, the cards expands, and if I'm between two cards, then any card comes up. We need a work around for this.

Thank you!

File: src/client/index.html [hunk 1]
```diff
@@ -1,201 +1,291 @@
...
-              <li><a href="#">Blog</a></li>
-              <li class="first"><a href="#">Events</a></li>
-              <li><a href="#About">About Us</a></li>
-              <li class="last"><a href="#contact_form">Contact Us</a></li>
-            </ul>
-          </nav>
-        </div>
-      </header>
-      <!-- Sticky Events section goes here -->
+ <head>
```
Comment (manrajgrover) [inline src/client/index.html:3]: Why was this reduced by one space? @Slayslot

Comment (handasolo) [inline src/client/index.html:3]: Don't remember doing this.

**Comment by manrajgrover** [comment]
I think we can close this PR now.

**Comment by CuriousLearner** [comment]
Bug Resolved in PR #20 

Closing this now.

**Pull request closed without merging**